common/formatter/text.go

```diff
@@ -22,8 +22,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/cmaglie/pb"
```

Comment:
the fork is diverging from upstream, any chance we can contribute instead of forking?

Reply:
Upstream library went from 1.x to 2.x (with API change) I don't think this patch will be accepted.
Maybe we can port directly to 2.x. Anyway I'm not bound to this particular library, we can evaluate other options. The features I missed much are:

* the progress bar should delete itself when reaching 100% so we have a clean output
* it should have no race conditions! (especially if there is a "rendering" go routine that runs in parallel with the download)

Reply:
Thanks for the explanation, we can have a look at v2 or alternatives later I guess